options/locale/locale_en-US.ini

```
@@ -730,8 +730,12 @@ migrate.lfs_mirror_unsupported = Mirroring LFS objects is not supported - use 'g
 migrate.migrate_items_options = Authentication is needed to migrate items from a service that supports them.
 migrated_from = Migrated from <a href="%[1]s">%[2]s</a>
 migrated_from_fake = Migrated From %[1]s
+migrate.migrate = Migrate From %s
 migrate.migrating = Migrating from <b>%s</b> ...
 migrate.migrating_failed = Migrating from <b>%s</b> failed.
+migrate.GitHub.description = Migrating data from Github.com or Github Enterprise.
+migrate.Git.description = Migrating or Mirroring git data from Git services
+migrate.GitLab.description = Migrating data from GitLab.com or Self-Hosted repository.
```

It would be better to use `.Name()` not `.Title()` for translations:
```suggestion
migrate.github.description = Migrating data from Github.com or Github Enterprise.
migrate.git.description = Migrating or Mirroring git data from Git services
migrate.gitlab.description = Migrating data from GitLab.com or Self-Hosted repository.
```